commands/diagnostics.vegman

```diff
@@ -119,3 +119,57 @@ function cmd_traceroute {
 function cmd_netstat {
   netstat "$@"
 }
+
+# D-Bus intrusion objects
+INTR_SERVICE="xyz.openbmc_project.intrusionsensor.service"
+INTR_GPIO_PIN="CASE_OPEN_FAULT_RST"
+INTR_DBUS_SERVICE="xyz.openbmc_project.IntrusionSensor"
+INTR_DBUS_PATH="/xyz/openbmc_project/Intrusion/Chassis_Intrusion"
+INTR_DBUS_INTERFACE="xyz.openbmc_project.Chassis.Intrusion"
+INTR_DBUS_PROPERTY="Status"
+
+# @sudo cmd_intrusion_on admin,tech
+# @restrict cmd_intrusion_on admin,tech
+# @doc cmd_intrusion_on
+# Turn the chassis intrusion detection on
+function cmd_intrusion_on {
+  sysctemctl start ${INTR_SERVICE}
+}
+
+# @sudo cmd_intrusion_off admin,tech
+# @restrict cmd_intrusion_off admin,tech
+# @doc cmd_intrusion_off
+# Turn the chassis intrusion detection off
+function cmd_intrusion_off {
+  sysctemctl stop ${INTR_SERVICE}
+}
+
+# @sudo cmd_intrusion_show admin,tech
+# @restrict cmd_intrusion_show admin,tech
+# @doc cmd_intrusion_show
+# Show the chassis intrusion detection status
+function cmd_intrusion_show {
+  state=$(busctl get-property ${INTR_DBUS_SERVICE} "${INTR_DBUS_PATH}" \
+          ${INTR_DBUS_INTERFACE} ${INTR_DBUS_PROPERTY} \
+          | awk '{print $NF;}' 2>/dev/null ||:)
```

Comment:
stderr redirection should be applied for busctl call not for awk. ;)

Reply:
Oops :D